main-page.html

```diff
@@ -0,0 +1,82 @@
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="style.css">
+    <title>HTML Tutorijal</title>
+</head>
+<body>
+<div class="container">
+    <div class="header">
+        <nav>
+            <ul class="navigations">
+                <li><a href="#" id="click-html">HTML</a></li>
+                <li><a href="#" id="click-css">CSS</a></li>
+                <li><a href="#" id="click-javascript">JAVASCRIPT</a></li>
+            </ul>
+        </nav>
+    </div>
+
+    <ul id="css-lists" class="languages-lists">
+        <li><button>CSS-grid</button></li>
+        <li><button>CSS-flex</button></li>
+    </ul>
+
+    <ul id="html-lists" class="languages-lists">
+        <li><button>Calendar</button></li>
+        <li><button>Form</button></li>
+        <li><button>Comments</button></li>
+    </ul>
+    <ul id="javascript-lists" class="languages-lists">
+        <li id="odd-numbers"><button>Odd Numbers</button></li>
+        <li id="even-numbers"><button>Even Numbers</button></li>
+        <li id="simple-prompt"><button>Simple Prompt</button></li>
+    </ul>
+
+
+</div>
+
+<script>
+
+    let clickHTML = document.getElementById('click-html');
+    let htmlLists = document.getElementById('html-lists');
+    clickHTML.addEventListener('click',openHTML);
+
+    function openHTML() {
+        htmlLists.style.display = 'block';
+
+    }
+
+    let clickCSS = document.getElementById('click-css');
+    let cssLists = document.getElementById('css-lists');
+    clickCSS.addEventListener('click',openCSS);
+
+    function openCSS() {
+        cssLists.style.display = 'block';
+        return cssLists
+    }
+
+    let clickJS = document.getElementById('click-javascript');
+    let jsLists = document.getElementById('javascript-lists');
+    clickJS.addEventListener('click',openJS);
```

Comment:
clickJS is used just once and can be written inline:
`document.getElementById('click-javascript').addEventListener('click',openJS);